data/base-spells.yaml

```diff
@@ -309,6 +309,13 @@ barb_abilities:
     activated_message: ^You .* to meditate
     expired_message: Focus meditation slips away from your mind
 
+paladin_abilities:
```

Comment:
Do you plan to update https://github.com/rpherbig/dr-scripts/blob/master/glyph-of-mana.lic to use this data?

Reply:
Not really, this should negate the need for that script.